BM/sdsi/tests

```diff
@@ -1,5 +1,8 @@
 # This file collects basic cases which can run against sdsi driver
 # on Intel® Architecture-based server platforms.
+# @hw_dep: general_test.sh -t fms -p "sdsi_fms_list" @ CPU Model doesn't support sdsi
```

Comment:
Hi @qwang59 ,  I don't think this is correct to call the script general_test.sh.  The parameter '-p' likely can not accept a file.  Could you please check it out? Or I missed something about it..

Reply:
I get it. Thanks.